mlflow/models/evaluation/base.py

```diff
@@ -35,6 +35,18 @@
 _logger = logging.getLogger(__name__)
 
 
+class MODEL_TYPE:
+    REGRESSOR = "regressor"
+    CLASSIFIER = "classifier"
+
+    def __init__(self):
+        raise NotImplementedError("This class is not meant to be instantiated.")
+
+    @classmethod
+    def get_all(cls):
+        return (cls.REGRESSOR, cls.CLASSIFIER)
```

Comment:
Do we want this to be a tuple instead of a list? Though I don't think it matters for now :D

Reply:
Good question. This is just my personal preference. A list also works, but I prefer a tuple because we don't mutate the result of this function.